edxval/models.py

```diff
@@ -600,6 +600,13 @@ class TranscriptPreference(TimeStampedModel):
         blank=True,
     )
     preferred_languages = ListField(verbose_name='Preferred Languages', max_items=50, default=[], blank=True)
+    video_source_language = models.CharField(
+        verbose_name='Video Source Language',
+        max_length=20,
```

Comment:
we should follow the same convention for language codes -- I think [this](https://github.com/edx/edx-val/pull/100/files#diff-bf00c7cdc19274a54f59d0526f065c8dR409) is what we decided in the first place. cc @mushtaqak